pages/reference/supervisor/bandwidth-reduction.md

```diff
@@ -4,52 +4,47 @@ title: Reduce bandwidth usage
 
-`BALENA_SUPERVISOR_LOG_CONTROL`: Any logs written by the user container or the device Agent are not sent to the dashboard when this variable is set to false.
+Your bandwidth measurements may vary based on hardware, location or cellular provider, but we expect them to be comparable. Of course your own app may add to network use. We encourage you to measure with your own device and workload! It's easy to deploy the Network Metrics Logger.
 
-`BALENA_SUPERVISOR_HARDWARE_METRICS`: [Device metrics][device-metrics] such as CPU temperature and memory usage won't be reported to the API, and won't be displayed on the summary page of the device dashboard.
+We wrote Network Metrics Logger to provide a generic, client-side tool to measure bandwidth. It uses the NodeJS system information [library](https://systeminformation.io/), which in turn reads network statistics from `/sys/class/net/{iface}/statistics`. Twilio provides a nice interface to their server-side statistics, but we wanted a tool that works across providers.
 
-Data usage impact
------------------
 
-| Service                                                                 | Usage (Including DNS overhead)                          | Related Configuration Variable       |
-|-------------------------------------------------------------------------|---------------------------------------------------------|--------------------------------------|
-| API poll (Once per 15min)                                               | 6650 Bytes per request                                  | BALENA_SUPERVISOR_POLL_INTERVAL      |
-| {{ $names.company.lower }} Supervisor update poll (Once every 24 hours) | 6693 Bytes per request                                  | Not configurable via variables       |
-| cloudlink enabled                                                       | 43 Bytes / second                                       | BALENA_SUPERVISOR_VPN_CONTROL        |
-| TCP check cost (When cloudlink disabled)                                | 47.36 Bytes / second                                    | BALENA_SUPERVISOR_CONNECTIVITY_CHECK |
-| Hardware metrics reporting                                              | ~66 Bytes every 10 seconds when there have been changes | BALENA_SUPERVISOR_HARDWARE_METRICS   |
+## Side Effects / Warnings
 
-Example minimum bandwidth settings
-----------------------------------
+**Cloudlink** Provides the ability to send instantaneous updates to the device. Turning off cloudlink has several downsides: (1) configuration and variable updates are delayed by the device API polling interval; (2) CLI access is disabled via the web terminal or SSH; (3) device's public URL is not accessible. To be clear, you can re-enable Cloudlink via API or console after turning it off. However you must wait for the next API poll from the device before the link is re-established.
```

Comment:
```suggestion
**Cloudlink** allows the device to be instantly notified of state changes. Turning off Cloudlink has several consequences:

- CLI access via the web terminal or SSH is disabled.
- Updates to configuration, environment variables and other state are not picked up by the device immediately, but are delayed instead until the next time the device polls the API.
- The Device Public URL feature is not available.

You can re-enable Cloudlink via API or console after turning it off. The device will re-establish the link the next time it polls the API.
```

------

Also, what do you mean with "via API or console"? Can you please elaborate as I don't think it's clear to the reader.

Reply:
Just changed that to "dashboard", which works better for Cloudlink.